File: Author.java
import java.util.Date;

public class Author {
    private String name;
    private Date birthDate;
    private String countryOfOrigin;
    private String shortBib;
    private int numOfBooks;

    public Author(){} //default ctor

    public Author(String name, Date birthDate, String origin, String shortBib, int numOfBooks){ //ctor
        this.name = name;
        this.birthDate = birthDate;
        this.countryOfOrigin = origin;
        this.shortBib = shortBib;
        this.numOfBooks = numOfBooks;
    }

    //getters and setters

    public String getName() {
        return name;
    }
    public Date getBirthDate() {
        return birthDate;
    }  
    public String getCountryOfOrigin() {
        return countryOfOrigin;
    }
    public String getShortBib() {
        return shortBib;
    }
    public int getNumOfBooks() {
        return numOfBooks;
    }

    public void setName(String name) {
        this.name = name;
    }
    public void setBirthDate(Date birthDate) {
        this.birthDate = birthDate;
    }
    public void setCountryOfOrigin(String countryOfOrigin) {
        this.countryOfOrigin = countryOfOrigin;
    }
    public void setShortBib(String shortBib) {
        this.shortBib = shortBib;
    }
    public void setNumOfBooks(int numOfBooks) {
        this.numOfBooks = numOfBooks;
    }
}

File: model/Author.php
<?php

class Author {
    private $name;
    private $birthDate;
    private $countryOfOrigin;
    private $shortBib;
    private $numOfBooks;

    //ctor
    public function Author(&$name, &$birthDate, &$origin, &$shortBib, &$numOfBooks){ 
        $this->name = $name;
        $this->birthDate = $birthDate;
        $this->countryOfOrigin = $origin;
        $this->shortBib = $shortBib;
        $this->numOfBooks = $numOfBooks;
    }

    //getters and setters

    public function getName() {
        return $this->name;
    }
    public function getBirthDate() {
        return $this->birthDate;
    }  
    public function getCountryOfOrigin() {
        return $this->countryOfOrigin;
    }
    public function getShortBib() {
        return $this->shortBib;
    }
    public function getNumOfBooks() {
        return $this->numOfBooks;
    }

    public function setName(&$name) {
        $this->name = $name;
    }
    public function setBirthDate(&$birthDate) {
        $this->birthDate = $birthDate;
    }
    public function setCountryOfOrigin(&$countryOfOrigin) {
        $this->countryOfOrigin = $countryOfOrigin;
    }
    public function setShortBib(&$shortBib) {
        $this->shortBib = $shortBib;
    }
    public function setNumOfBooks(&$numOfBooks) {
        $this->numOfBooks = $numOfBooks;
    }
}

?>